### PR TITLE
add documentation for the new rollback command

### PR DESCRIPTION
**Pull request by maxdml**

### CODE REVIEW

File: docs/cloud-tutorials/database-management.md [hunk 1]
```diff
@@ -60,19 +60,21 @@ To manage your applications' database schemas, you must define schema migrations
...
   migrate: ['npx knex migrate:latest']
+  rollback: ['npx knex migrate:down']
 ```
 
-To run your migrations locally, run `npx dbos-sdk migrate`.
+To run your migrations locally, run `npx dbos-sdk migrate` or `npx dbos-sdk rollback`.
 
-When you [deploy](./application-management.md#registering-and-deploying-applications) an application to DBOS Cloud, it runs `npx dbos-sdk migrate` to apply all schema changes before starting your application.
+When you [deploy](./application-management.md#registering-and-deploying-applications) or [rollback](./application-management.md#rolling-back-applications) a DBOS Cloud application, it runs `npx dbos-sdk migrate` or `npx dbos-sdk rollback`, respectively, to apply all schema changes before starting your application.
+Note that database schema migration should be backward compatible with the schema used by your currently running workflows.
```
Comment (kraftp) [inline docs/cloud-tutorials/database-management.md:77]: This is a very subtle point that needs to be separated into an infobox and explained.  For example:

:::info
When you deploy a new version of an application, DBOS Cloud first runs schema migrations, then upgrades executors to use the new application version. Thus, you should be careful making breaking schema changes such as deleting or renaming a column that are not compatible with the previous application version.
:::

File: docs/cloud-tutorials/application-management.md [hunk 2]
```diff
@@ -57,6 +57,13 @@ If you edit your application or schema, run `npx dbos-cloud app deploy` again to
 You don't have to worry about changing database server connection parameters like `hostname` or `password` in [`dbos-config.yaml`](../api-reference/configuration.md) to deploy an application to the cloud&#8212;DBOS automatically applies the connection information of your cloud database instance.
 :::
 
+When you deploy a new version of your application to DBOS cloud, currently running workflows will still run to completion. This means database schema migrations must be backward compatible with these running workflows.
+
+### Rolling back applications
+
+To trigger the execution of [database rollback commands](./database-management.md#database-schema-management), simply run `npx dbos-cloud app rollback`. DBOS cloud will re-deploy your application and trigger your database rollback commands, rather than the migrate commands.
+Note that schema migration must be backward compatible with the database schema your currently pending workflows are expecting.
```
Comment (kraftp) [inline docs/cloud-tutorials/application-management.md:65]: What would a backwards-compatible rollback even look like?

File: docs/cloud-tutorials/application-management.md [hunk 3]
```diff
@@ -57,6 +57,13 @@ If you edit your application or schema, run `npx dbos-cloud app deploy` again to
 You don't have to worry about changing database server connection parameters like `hostname` or `password` in [`dbos-config.yaml`](../api-reference/configuration.md) to deploy an application to the cloud&#8212;DBOS automatically applies the connection information of your cloud database instance.
 :::
 
+When you deploy a new version of your application to DBOS cloud, currently running workflows will still run to completion. This means database schema migrations must be backward compatible with these running workflows.
+
+### Rolling back applications
+
+To trigger the execution of [database rollback commands](./database-management.md#database-schema-management), simply run `npx dbos-cloud app rollback`. DBOS cloud will re-deploy your application and trigger your database rollback commands, rather than the migrate commands.
```
Comment (kraftp) [inline docs/cloud-tutorials/application-management.md:64]: This isn't clear--break it down more.

File: docs/cloud-tutorials/application-management.md [hunk 4]
```diff
@@ -57,6 +57,13 @@ If you edit your application or schema, run `npx dbos-cloud app deploy` again to
 You don't have to worry about changing database server connection parameters like `hostname` or `password` in [`dbos-config.yaml`](../api-reference/configuration.md) to deploy an application to the cloud&#8212;DBOS automatically applies the connection information of your cloud database instance.
 :::
 
+When you deploy a new version of your application to DBOS cloud, currently running workflows will still run to completion. This means database schema migrations must be backward compatible with these running workflows.
```
Comment (kraftp) [inline docs/cloud-tutorials/application-management.md:60]: Same comment as below, set this off into an infobox with more explanation.

Comment (maxdml) [reply docs/cloud-tutorials/application-management.md:60]: Should be addressed in both cases

**Review by kraftp — APPROVED** [review]

**Pull request merged**